lib/src/arithmetic/floating_point/floating_point_multiplier_simple.dart

```diff
@@ -91,12 +90,12 @@ class FloatingPointMultiplierSimple extends FloatingPointMultiplier {
     final isInfLatch = localFlop(isInf);
     final isNaNLatch = localFlop(isNaN);
 
-    final leadingOnePosPre = ParallelPrefixPriorityEncoder(mantissa.reversed,
-            ppGen: ppTree, name: 'leading_one_encoder')
-        .out
-        .named('leadingOneRaw')
-        .zeroExtend(exponentWidth + 2)
-        .named('leadingOneRawExtended', naming: Naming.mergeable);
+    final leadingOnePosPre =
+        priorityGen(mantissa.reversed, name: 'leading_one_encoder')
```

Comment:
not using the `valid` here?  should it not be required then in the argument, or should it have been used?